src/create.js

```diff
@@ -1,38 +1,56 @@
+import post from './module/post';
+
 export default function Scores() {
-  const submitButton = document.querySelector('.submitBtn');
+  const refreshBtn = document.querySelector('.refreshBtn');
   const nameInput = document.querySelector('.name');
   const scoreInput = document.querySelector('.score');
+  const Form = document.querySelector('form');
 
-  const addToList = () => {
-    const scores = JSON.parse(localStorage.getItem('scores')) || [];
-    const objectScores = {
-      name: nameInput.value,
-      score: scoreInput.value,
-    };
-    scores.push(objectScores);
-    localStorage.setItem('scores', JSON.stringify(scores));
-    window.location.reload();
-  };
+  const api = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/2ehdutnghjd400irjf8nnv/scores/';
 
-  const display = () => {
-    const scores = JSON.parse(localStorage.getItem('scores')) || [];
-    scores.forEach((score) => {
-      const scoreList = document.querySelector('.scores-div');
+  const display = (data) => {
+    const scoreList = document.querySelector('.scores-div');
+    data.forEach((person) => {
       const list = document.createElement('li');
       list.classList.add('list');
       list.innerHTML = `
         <div class="one">
-        <p>${score.name}: ${score.score}</p>
+        <p>${person.user}: ${person.score}</p>
         </div>
        `;
       scoreList.appendChild(list);
     });
   };
-  display();
-  submitButton.addEventListener('click', () => {
-    if (nameInput.value && scoreInput.value === '') return;
-    addToList(nameInput.value, scoreInput.value);
-    nameInput.value = '';
-    scoreInput.value = '';
+
+  Form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const data = {
+      user: nameInput.value,
+      score: scoreInput.value,
+    };
+    if (nameInput.value && scoreInput.value === '');
+    post(api, data);
+    Form.reset();
+  });
+
+  const Get = async (api) => {
+    try {
+      const response = await fetch(api, {
+        method: 'GET',
+        headers: {
+          'content-Type': 'application/json',
+        },
+      });
+      const data = await response.json();
+      const finalResults = data.result;
+      display(finalResults);
+      return data;
+    } catch (error) {
+      return error;
+    }
+  };
+  refreshBtn.addEventListener('click', (e) => {
+    Get(api);
+    e.stopPropagation();
   });
-}
\ No newline at end of file
+}
```

Comment:
- [ ] Good job with your project, it almost works as expected as once the Refresh button is clicked, it hits the GET API endpoint causing the scores list to change. The issue is that for each `Click action`(on the Refresh button), it generates new a list by repeating the existing one and adding a new one i.e the scores list is repeated according to the number of times you click the `Refresh` button. Below is an example of that issue:

| One Clik | Two clicks |
| ------------ | ---------------|
| ![Screenshot from 2022-08-26 02-18-39](https://user-images.githubusercontent.com/83042742/186790971-87c42064-cd83-4069-8123-bb891b870242.png) | ![Screenshot from 2022-08-26 02-19-13](https://user-images.githubusercontent.com/83042742/186790992-fcafaddf-7e58-47bb-be87-2109175345e2.png) |